main.py

```diff
@@ -44,6 +49,34 @@ def tba_webhook():
     return msg_type
 
 
+@app.route('/predictions')
+def give_predictions():
+    return str(trueskill_predictions)
+
+
+@app.route('/team_trueskill/<team>')
+def team_trueskill(team):
+    return str(trueskill.skill(team))
+
+
+@app.route('/event_trueskill_ranking/<event>')
+def event_trueskil(event):
+    event_teams = trueskill.get_teams_at_event(event)
+    skills = [(team['key'], trueskill.skill(team['key']), team['nickname']) for team in event_teams]
+    skills = sorted(skills, key=lambda skill: skill[1], reverse=True)
+    msg = ''
+    num = 0
+    for skill in skills:
+        num += 1
+        msg += '<li>%s.  %s - %s - %0.1f\n</li> \n' % (str(num), skill[0][3:], skill[2], skill[1])
```

Comment:
To clarify: generating HTML using format strings/string concatenation without ensuring everything is HTML escaped is a huge no-no.

Either use a templating engine that does HTML escaping, or return a JSON blob and do this client-side (properly).